packages/cli/src/spec/configSpec.ts

```diff
@@ -22,7 +22,7 @@ import {loadOrCreateConfig} from '../lib/config';
 import * as mock from 'mock-fs';
 import * as inquirer from 'inquirer';
 
-const DEFAULT_CONFIG_FOLDER = join(homedir(), '.bubblewrap-config');
+const DEFAULT_CONFIG_FOLDER = join(homedir(), '.bubblewrap');
 const DEFAULT_CONFIG_NAME = 'bubblewrap-config.json';
```

Comment:
Maybe we should just change the file name to `~/.bubblewrap/config.json`, `~/.bubblewrap/bubblewrap-config.json` is a bit tautological.

Reply:
Yeah, makes sense. Renamed.